GherkinSyncTool/GherkinSyncTool/Synchronizers/TestRailSynchronizer/TestRailSynchronizer.cs

```diff
@@ -37,23 +38,21 @@ public void Sync(List<IFeatureFile> featureFiles)
                 {
                     var tagId = scenario.Tags.FirstOrDefault(tag => Regex.Match(tag.Name, config.TagIdPattern, RegexOptions.IgnoreCase).Success);
 
+                    var createCaseRequest = _caseContentBuilder.BuildCreateCaseRequest(scenario, featureFile);
```

Comment:
I am not sure that it is correct naming policy - I propose to rename the variable to **caseRequest** and method to **BuildCreateOrUpdateCaseRequest**

Reply:
Fixed